tardis/transport/montecarlo/base.py

```diff
@@ -120,9 +114,8 @@ def initialize_transport_state(
 
         geometry_state = simulation_state.geometry.to_numba()
 
-        opacity_state = self.opacity_solver.solve(plasma)
         opacity_state_numba = opacity_state_to_numba(
-            opacity_state, self.opacity_solver.line_interaction_type
+            opacity_state, self.line_interaction_type
         )
         opacity_state_numba = opacity_state_numba[
             simulation_state.geometry.v_inner_boundary_index : simulation_state.geometry.v_outer_boundary_index
```

Comment:
Suggests we perhaps need some slice handling by the solver? Or something similar. Not directly relevant to this PR.